crates/proc_macro_srv/src/lib.rs

```diff
@@ -56,6 +66,11 @@ impl ProcMacroSrv {
                 None => env::remove_var(k),
             }
         }
+        if let Some(dir) = prev_working_dir {
+            if let Err(err) = std::env::set_current_dir(dir) {
+                eprint!("Failed to set the current working dir to {:?}. Error: {:?}", dir, err)
```

Comment:
```suggestion
                eprint!("Failed to set the current working dir to {}. Error: {:?}", dir, err)
```

, I think.

Reply:
PathBuf does not implement Display. I'm not sure how best to get around this

Reply:
`dir.display()`, I think